builtin/mainmenu/content/contentdb.lua

```diff
@@ -129,6 +145,15 @@ local function start_install(package, reason)
 					end
 					conf:set("author",     package.author)
 					conf:set("release",    package.release)
+					if package.aliases then
+						local gameid_alias = ""
+						for _, alias in pairs(package.aliases) do
```

Comment:
Nitpick: Why `pairs` here but `ipairs` below? I'd suggest using `ipairs` in both cases since you're expecting this to be a list (or using `pairs` since in both cases you don't care about order). But it should be consistent.